Laravel-Backpack/demo
# Add PAN Analytics add-on to demo, for product analytics

## WHY

### BEFORE - What was wrong? What was happening before this PR?

Pedro had created [an addon for PAN Analytics](https://github.com/Laravel-Backpack/pan-panel) but we weren't showing it anywhere.

### AFTER - What is happening after this PR?

We are showing it in our Demo, and tracking what people click on our Welcome page and our Menu:

![CleanShot 2024-10-30 at 17 08 47@2x](https://github.com/user-attachments/assets/2b8fb6b6-3d64-4dcc-97f0-15ba4436f322)



## HOW

### How did you achieve that, in technical terms?

Install package and add `data-pan=name` in a few places in our custom blade files.


### Is it a breaking change or non-breaking change?

Non-breaking. BUT. The `composer.json` needs to change once the add-on is released, to load the first major version.

### How can we test the before & after?

Todo:
- Switch to this branch of the demo. 
- Run `composer install` and `php artisan migrate`. 
- Then play around in the demo to load up the analytics db with some entries.
- Then go to Addons > Analytics to see the results.
